### ISSUE TITLE
Exporting word list metadata misses some languages?

### ISSUE THREAD

**Issue by seannyD** [clld/asjp#11]
If you go to https://asjp.clld.org/languages, click on download, then csv, you get a useful list of languages matched to glottocodes, iso codes etc.  I assumed this would download all the data, but it seems like it misses some languages.  For example, Latin is listed in the web table, but does not appear in the csv. But when I filter the web table for Latin, and then export, it does appear in the csv.

Basically, I was looking for a mapping from ASJP ID to glottocode, and that seemed quite hard to get. I ended up downloading the formatted database from the ASJP-data repository.

**Comment by xrotwang** [comment]
The [download page](https://asjp.clld.org/download) also links to the zipped [CLDF dataset](https://cdstar.shh.mpg.de/bitstreams/EAEA0-CFA9-F536-516C-0/asjp_dataset.cldf.zip) (on the right side). The file `languages.csv` in there should be all you need.

The problem you ran into with the download links from the web app tables, is probably the 2000 item limit imposed on those for performance reasons.

So I guess, we at least got from "it's hard to get" to "it's hard to find" :)
